pages/linux/yay.md

```diff
@@ -0,0 +1,35 @@
+# yay
+
+> Yet Another Yogurt - A utility for Arch Linux to build and install packages from the Arch User Repository.
+
+- Search and install packaged from the repos and the AUR using yogurt mode:
```

Comment:
>packaged

Typo

>yogurt mode

I guess the man page does call it this. I prefer "interactive search and install" myself and may even change it in the man page at some point.

I'd go for "Interactivity search and install packages from the repos and AUR". And if you really want to you could tack on (yogurt mode).